### ISSUE TITLE
LG StandbyMe touch screen support

### ISSUE THREAD

**Issue by lvntruong** [webosbrew/youtube-webos#38]
Hi team,
Currently unable to use the touch screen on app - LG StandbyMe devices.
Thanks!

**Comment by Julmik647** [comment]
I don't think that LG StandbyMe devices are popular enough to add touchscreen support for this app on them.
